src/selector.js

```diff
@@ -50,11 +42,17 @@ const SelectorListWrapper = styled.div`
   opacity: 0;
   display: flex;
 
-  .is-open & {
-    pointer-events: auto;
-    opacity: 1;
-    transition: opacity 0.5s ease;
-  }
+  ${({ isOpen }) => {
+	if (isOpen) {
+	  return `
```

Comment:
this should use the `css` function from styled-components https://www.styled-components.com/docs/api#css

```suggestion
	  return css`
```

Reply:
since this is just a string, I don't think we need to run it through the css function